extracting_article_content_from_HTML.py

```diff
@@ -0,0 +1,40 @@
+# Extract article content from HTML with Boilerpipe
+# Be sure to install boilerpipe (pip install boilerpipe)
```

Comment:
Input, and output? The script, like most of our HOW-TOs is fully self-contained. It downloads some source HTML files from S3 and then extracts the textual content into an SFrame. Not quite clear what you're looking for. A description of the boilerpipe module? How pip works?